package.json

```diff
@@ -25,132 +25,133 @@
-    "@babel/core": "^7.0.1",
-    "@babel/plugin-proposal-class-properties": "^7.0.0",
-    "@babel/plugin-proposal-decorators": "^7.0.0",
-    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
-    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
-    "@babel/plugin-transform-react-inline-elements": "^7.0.0",
-    "@babel/plugin-transform-react-jsx-self": "^7.0.0",
-    "@babel/plugin-transform-react-jsx-source": "^7.0.0",
-    "@babel/plugin-transform-runtime": "^7.0.0",
-    "@babel/preset-env": "^7.0.0",
+    "@babel/core": "^7.2.2",
+    "@babel/plugin-proposal-class-properties": "^7.2.3",
+    "@babel/plugin-proposal-decorators": "^7.2.3",
+    "@babel/plugin-proposal-object-rest-spread": "^7.2.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
+    "@babel/plugin-transform-react-inline-elements": "^7.2.0",
+    "@babel/plugin-transform-react-jsx-self": "^7.2.0",
+    "@babel/plugin-transform-react-jsx-source": "^7.2.0",
+    "@babel/plugin-transform-runtime": "^7.2.0",
+    "@babel/preset-env": "^7.2.3",
     "@babel/preset-react": "^7.0.0",
-    "@babel/runtime": "^7.0.0",
+    "@babel/runtime": "^7.2.0",
+    "@gfx/zopfli": "^1.0.10",
     "array-includes": "^3.0.3",
-    "autoprefixer": "^8.6.5",
-    "axios": "~0.16.2",
+    "autoprefixer": "^9.4.3",
+    "axios": "^0.18.0",
     "babel-core": "^7.0.0-bridge.0",
-    "babel-loader": "^8.0.2",
+    "babel-loader": "^8.0.4",
     "babel-plugin-lodash": "^3.3.4",
     "babel-plugin-preval": "^3.0.1",
-    "babel-plugin-react-intl": "^3.0.0",
-    "babel-plugin-transform-react-remove-prop-types": "^0.4.15",
+    "babel-plugin-react-intl": "^3.0.1",
+    "babel-plugin-transform-react-remove-prop-types": "^0.4.21",
+    "babel-runtime": "^6.26.0",
     "classnames": "^2.2.5",
     "compression-webpack-plugin": "^2.0.0",
     "cross-env": "^5.1.4",
-    "css-loader": "^1.0.0",
+    "css-loader": "^2.1.0",
+    "cssnano": "^4.1.8",
     "detect-passive-events": "^1.0.2",
-    "dotenv": "^4.0.0",
+    "dotenv": "^6.2.0",
     "emoji-mart": "Gargron/emoji-mart#build",
     "es6-symbol": "^3.1.1",
     "escape-html": "^1.0.3",
     "exif-js": "^2.3.0",
-    "express": "^4.16.2",
-    "file-loader": "^2.0.0",
+    "express": "^4.16.4",
+    "fibers": "^3.1.1",
+    "file-loader": "^3.0.1",
     "font-awesome": "^4.7.0",
     "glob": "^7.1.1",
     "history": "^4.7.2",
-    "http-link-header": "^0.8.0",
+    "http-link-header": "^1.0.2",
     "immutable": "^3.8.2",
     "imports-loader": "^0.8.0",
-    "intersection-observer": "^0.5.0",
+    "intersection-observer": "^0.5.1",
     "intl": "^1.2.5",
     "intl-messageformat": "^2.2.0",
     "intl-relativeformat": "^2.1.0",
     "is-nan": "^1.2.1",
     "js-yaml": "^3.11.0",
-    "lodash": "^4.17.5",
+    "lodash": "^4.7.11",
     "mark-loader": "^0.1.6",
-    "marky": "^1.2.0",
-    "mini-css-extract-plugin": "^0.4.2",
+    "marky": "^1.2.1",
+    "mini-css-extract-plugin": "^0.5.0",
     "mkdirp": "^0.5.1",
-    "node-sass": "^4.9.2",
```

Comment:
Replace to `sass`.

Because `node-sass` depends on C ++, it takes time to install the package. Since `sass` is provided with pure JavaScript, it can be installed immediately.